implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java

```diff
@@ -270,22 +349,24 @@ Stream<String> toDistributionSummaryLine(DistributionSummary meter) {
             return Stream.empty();
         }
 
-        return createSummaryLine(meter, snapshot.getMin(), snapshot.getMax(), snapshot.getTotal(), snapshot.getCount());
+        return createSummaryLine(meter, seenMetadata, snapshot.getMin(), snapshot.getMax(), snapshot.getTotal(),
+                snapshot.getCount());
     }
 
-    Stream<String> toLongTaskTimerLine(LongTaskTimer meter) {
-        return toSummaryLine(meter, meter.takeSnapshot(), getBaseTimeUnit());
+    Stream<String> toLongTaskTimerLine(LongTaskTimer meter, Map<String, String> seenMetadata) {
+        return toSummaryLine(meter, seenMetadata, meter.takeSnapshot(), getBaseTimeUnit());
     }
 
-    Stream<String> toTimeGaugeLine(TimeGauge meter) {
-        return toMeterLine(meter, this::createGaugeLine);
+    Stream<String> toTimeGaugeLine(TimeGauge meter, Map<String, String> seenMetadata) {
+        return toMeterLine(meter, (theMeter, measurement) -> createGaugeLine(theMeter, seenMetadata, measurement));
     }
 
-    Stream<String> toFunctionCounterLine(FunctionCounter meter) {
-        return toMeterLine(meter, this::createCounterLine);
+    Stream<String> toFunctionCounterLine(FunctionCounter meter, Map<String, String> seenMetadata) {
+        return toMeterLine(meter, (theMeter, measurement) -> createCounterLine(theMeter, seenMetadata, measurement));
+        // return toMeterLine(meter, this::createCounterLine);
```

Comment:
looks like this is a leftover :slightly_smiling_face:

Reply:
Good eye!